robot/Cumulus/resources/NPSPSettingsPageObject.py

```diff
@@ -54,4 +58,12 @@ def wait_until_advanced_mapping_is_enabled(self):
             else:
                 raise AssertionError(
                     "Timed out waiting for Advanced Mapping is enabled to display"
-                )
\ No newline at end of file
+                )
+                
+    def click_configure_advanced_mapping(self):
+        """clicks on Configure Advanced Mapping and waits for Manage Advanced Mapping page to load and loads the page object for that page"""
+        locator=npsp_lex_locators['id'].format("navigateAdvancedMapping")
+        self.selenium.click_element(locator)
+        self.npsp.wait_until_url_contains("c__BDI_ManageAdvancedMapping")   
+        self.pageobjects.load_page_object("Custom", "ManageAdvancedMapping") 
```

Comment:
You can modify it to _self.pageobjects.current_page_should_be("Custom", "ManageAdvancedMapping")_.
This will call the "_is_current_page" method of that page object and then load the page object too.